dessert/conf.py

```diff
@@ -0,0 +1,2 @@
+DISABLE_INTROSPECTION_KEY = 'disable_introspection'
```

Comment:
@maormarcus I really prefer this to be an object with specific methods to enable/disable settings. If that is implemented so all queries of the sort "is X enabled" should be also exported as functions